### DISCUSSION TITLE
ETH needed to enable Mainnet for Custom JWT verifier deployment?

### DISCUSSION THREAD

**Discussion by peerwaya** [Web3Auth/Web3Auth#410] [Help]
I want to enable mainnet for a custom JWT verifier. Do I need ETH? It keeps  saying processing

**Comment by himanshuchawla009** [comment]
You can just submit the request to deploy the verifier from https://dashboard.web3auth.io. Our admins will approve the request and deploy a verifier for you on mainnet. (i.e  You don't need to pay for Ethereum gas fee for verifier deployment, this is being paid by web3auth.)

**Comment by peerwaya** [comment]
I have done that already. Although is shows status is failed.
``` Something happened during deployment. Our team is looking into it and will be in touch with you via email```

**Reply by himanshuchawla009** [reply]
@tainguyentt , plz help here.

**Reply by tanguyenvn** [reply]
@peerwaya could you give the name of your custom verifier that you submitted so that I can check?

**Reply by peerwaya** [reply]
@tainguyentt `gotok`  is the name of the custom verifier for mainnet
